src/core/components/model-example.jsx

```diff
@@ -61,39 +62,58 @@ export default class ModelExample extends React.Component {
-        </li>
-        { schema ? <li className={ "tabitem" + ( this.state.activeTab === "model" ? " active" : "") }>
-          <a className={ "tablinks" + ( isExecute ? " inactive" : "" )} data-name="model" onClick={ this.activeTab }>
-            {isOAS3 ? "Schema" : "Model" }
-          </a>
-        </li> : null }
-      </ul>
-      <div>
-        {
-          this.state.activeTab === "example" ? (
-            example ? example : (
+    return (
+      <div className="model-example">
+        <ul className="tab" role="tablist">
+          <li className={cx("tabitem", { "active": this.state.activeTab === "example" })} role="presentation">
+            <button
+              aria-selected={this.state.activeTab === "example"}
+              className="tablinks"
+              data-name="example"
+              onClick={ this.activeTab }
+              role="tab"
+            >
+              {isExecute ? "Edit Value" : "Example Value"}
+            </button>
+          </li>
+          { schema && (
+            <li className={cx("tabitem", { "active": ["model", "schema"].includes(this.state.activeTab) })} role="presentation">
+              <button
+                aria-selected={["model", "schema"].includes(this.state.activeTab)}
+                className={ "tablinks" + ( isExecute ? " inactive" : "" )}
+                data-name={isOAS3 ? "schema" : "model" }
+                onClick={ this.activeTab }
+                role="tab"
+              >
+                {isOAS3 ? "Schema" : "Model" }
+              </button>
+            </li>
+          )}
+        </ul>
+        {this.state.activeTab === "example" && (
+          <div data-name="examplePanel" role="tabpanel" aria-hidden={this.state.activeTab !== "example"} tabIndex="0">
+            {example ? example : (
               <HighlightCode value="(no example available)" getConfigs={ getConfigs } />
-            )
-          ) : null
-        }
-        {
-          this.state.activeTab === "model" && <ModelWrapper schema={ schema }
-                                                     getComponent={ getComponent }
-                                                     getConfigs={ getConfigs }
-                                                     specSelectors={ specSelectors }
-                                                     expandDepth={ defaultModelExpandDepth }
-                                                     specPath={specPath}
-                                                     includeReadOnly = {includeReadOnly}
-                                                     includeWriteOnly = {includeWriteOnly}/>
+            )}
+          </div>
+        )}
 
-
-        }
+        {["model", "schema"].includes(this.state.activeTab) && (
+          <div data-name={isOAS3 ? "schemaPanel" : "modelPanel" } role="tabpanel" aria-hidden={this.state.activeTab === "example"} tabIndex="0">
```

Comment:
Instead of using data attributes here, wouldn't be more idiomatic to use `aria-labelledby` attribute?

Reply:
Right let's leave data attributes to decouple the testing from the JSX structure.